### PR TITLE
Example Change, Travis Change, Doc Change

### PR DESCRIPTION
**Pull request by nabobalis**
I have no idea how to label this without adding everything.

### CODE REVIEW

**Comment by nabobalis** [comment]
The example now plays out like this:

Figure 1:
figure_1

Figure 2:
figure_1-1

**Comment by nabobalis** [comment]
Travis looks like:

screenshot from 2017-11-09 15-36-42

Cron jobs are:
screenshot from 2017-11-09 15-40-34

**Comment by prateekiiest** [comment]
Which example @nabobalis ?

**Comment by nabobalis** [comment]
@prateekiiest I merged the masking hot pixels and finding bright pixel into one

**Comment by prateekiiest** [comment]
Figure 1 was the previous case before merged?

**Comment by nabobalis** [comment]
Pretty much. The masking example did not actually mask anything. I forgot to fix it a while back. Now it masks but its not pretty code.

**Comment by prateekiiest** [comment]
ok, but its working

**Comment by nabobalis** [comment]
It is now.

File: examples/finding_masking_bright_pixels.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+###############################################################################
+# Now we create a new custom AIAMap with a mask around the brightest pixel.
+# We have to build two arrays which include all of the x and y pixel indices.
+# These indices are for the entire image.
+# We must not forget to add the correct units because we will next pass
+# into a SunPy function which all require them.
+x, y = np.meshgrid(*[np.arange(v.value) for v in aia.dimensions]) * u.pixel
+
+###############################################################################
+# Now we can convert this to helioprojective coordinates and create a new
```
Comment (Cadair) [inline examples/finding_masking_bright_pixels.py]: this text now seems out of sync with the rest of the example.

Comment (nabobalis) [reply examples/finding_masking_bright_pixels.py]: I thought i changed that. Seems not!

**Comment by Cadair** [comment]
@nabobalis what's the motivation for merging those three examples?

**Comment by nabobalis** [comment]
@Cadair One example was a subset of the other, so I thought might as well get rid of the subset example.

File: examples/finding_masking_bright_pixels.py (hunk 1 above)
Comment (nabobalis) [reply examples/finding_masking_bright_pixels.py]: well thats still correct to what i do in the code no?

**Comment by nabobalis** [comment]
Dont ask about test_timeseries*.py

File: CHANGELOG.rst [hunk 2]
```diff
@@ -9,6 +9,13 @@ Bug Fixes
 - Fix docstring formatting for net.vso.attrs [#2309]]
 - Fix coloring of ticks under matplotlib 2.0 default style [#2320]
 - Always index arrays with tuples in `ImageAnimator` [#2320]
+- Misc Doc fixes [#2317] [#2289]
+- Misc Readme Changes [#2281] [#2283]
```
Comment (Cadair) [inline CHANGELOG.rst]: This is not a useful changelog entry!

File: examples/brightness_pixel_location.py [hunk 3]
```diff
@@ -1,32 +0,0 @@
-"""
-============================
```
Comment (Cadair) [inline examples/brightness_pixel_location.py]: I think I am 👎 on removing this example. I know that it's basically the same as the other one, but it's a super simple reference example on over plotting coordinates.

File: examples/brightness_pixel_location.py [hunk 4]
```diff
@@ -1,32 +0,0 @@
...
-import sunpy.map
-from sunpy.data.sample import AIA_171_IMAGE
-
-###############################################################################
-# We first create the Map using the sample data and import the coordinate
-# functionality.
-aia = sunpy.map.Map(AIA_171_IMAGE)
-
-pixel_pos = np.argwhere(aia.data == aia.data.max())*u.pixel
-hpc_max = aia.pixel_to_data(pixel_pos[:, 1], pixel_pos[:, 0])
```
Comment (Cadair) [inline examples/brightness_pixel_location.py]: `pixel_to_world`

File: CHANGELOG.rst (hunk 2 above)
Comment (nabobalis) [reply CHANGELOG.rst]: YES THEY ARE

File: examples/brightness_pixel_location.py (hunk 3 above)
Comment (nabobalis) [reply examples/brightness_pixel_location.py]: FINE

File: examples/brightness_pixel_location.py (hunk 4 above)
Comment (nabobalis) [reply examples/brightness_pixel_location.py]: FINE

**Pull request merged**